backend/src/main/java/gov/cdc/usds/simplereport/service/UploadService.java

```diff
@@ -56,7 +57,9 @@ public void processPersonCSV(InputStream csvStream) throws IOException {
             final Map<String, String> row = valueIterator.next();
 
             final LocalDate patientDOB = LocalDate.parse(row.get("DOB"), DATE_FORMATTER);
-            _ps.addPatient(row.get("ID"),
+            _ps.addPatient(
+                    UUID.fromString(row.get("facilityId")),
```

Comment:
Bringing a DM discussion into the PR: since facility ID is something customer's won't know (especially during onboarding when they don't exist yet), should we support using the facility name in the spreadsheet to identify where the patient is going to be seen?

Also, should this be allowed to be null?

Reply:
The person upload is currently an artisanal upload and facility name or id have the same amount of manual work.   

> Also, should this be allowed to be null?
yes, updated